examples/2d/mesh2d_manual.rs

```diff
@@ -209,10 +209,10 @@ type DrawColoredMesh2d = (
 // using `include_str!()`, or loaded like any other asset with `asset_server.load()`.
 const COLORED_MESH2D_SHADER: &str = r"
 // Import the standard 2d mesh uniforms and set their bind groups
-#import bevy_sprite::mesh2d_view_bind_group
-[[group(0), binding(0)]]
-var<uniform> view: View;
-#import bevy_sprite::mesh2d_struct
+#import bevy_sprite::mesh2d_view_types
```

Comment:
What if we imported `X_types` inside of `X_bindings`? Every `X_bindings` import will need a corresponding `X_types` import to work. We could cut out a lot of boilerplate this way. We'd still have the granularity of only importing `X_types` when that is necessary, but I'd wager that 99% of cases just need the binding imports to work. 

This could simplify to:

```rust
#import bevy_sprite::mesh2d_view_bindings
#import bevy_sprite::mesh2d_bindings
```

(if we also move the Mesh2d binding to an import, per my other comment).

Reply:
I've verified that the nested import approach works.

Reply:
Done.